std/json.d

```diff
@@ -602,6 +602,46 @@ struct JSONValue
         assert(j.type == JSONType.object);
     }
 
+    static enum : JSONValue {
+        /**
+         * An enum value that can be used to obtain a `JSONValue` representing
+         * an empty JSON object.
+         */
+        emptyObject = JSONValue(string[string].init),
+
+        /**
+         * An enum value that can be used to obtain a `JSONValue` representing
+         * an empty JSON array.
+         */
+        emptyArray = JSONValue(JSONValue[].init)
+    }
```

Comment:
These don't need to be in an enum type.

```d
enum emptyObject = JSONValue(string[string].init);
enum emptyArray = JSONValue(JSONValue[].init);
```
With your current docs of course.

Reply:
Done! I have also separated the `unittest` block so that the tests relevant for the particular enum are immediately following it.